cpp/ps0001_soin.cpp

```diff
@@ -1,31 +1,27 @@
-﻿
 #include <iostream>
 
 using namespace std;
 
 int main()
 {
-    int A;
-    int B;
-    int C;
-
-
-        cout << "Please enter the first Number A ? " << endl;
-        cin >> A; 
-
-        cout << "Please enter the first Number B ? " << endl;
-        cin >> B;
-
-        cout << "Please enter the first Number C ? " << endl;
-        cin >> C;
-
-        cout << "-----------------------------------------------" << endl;
-
-        cout << A + B + C << endl;
+    // prompting user for 3 numbers to add 1/2
+    short num1;
+    short num2;
+    short num3;
+    
+    cout << "Please enter the first Number A ? " << endl;
+    cin >> num1; 
 
+    cout << "Please enter the first Number B ? " << endl;
+    cin >> num2;
 
+    cout << "Please enter the first Number C ? " << endl;
+    cin >> num3;

```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add